views/partials/password_tip_svg.hbs

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
random thought, but since this is now an inlined SVG, do we want this header (since it may be repeated 4-6 times within the HTML doc)?

Reply:
Pruned